<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qxgedit - Qt XG Editor.

   Copyright (C) 2005-2022, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qxgeditOptionsForm</class>
 <widget class="QDialog" name="qxgeditOptionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qxgedit.qrc">:/images/qxgedit.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="OptionsTabWidget">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralTabPage">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="ConfirmResetCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to ask for confirmation on parameter reset</string>
         </property>
         <property name="text">
          <string>Confirm parameter &amp;resets</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="8">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>8</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QCheckBox" name="ConfirmRemoveCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to ask for confirmation on removals</string>
         </property>
         <property name="text">
          <string>&amp;Confirm removals</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="CompletePathCheckBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Whether to show the complete directory path of loaded session files</string>
         </property>
         <property name="text">
          <string>Show complete &amp;path of session files</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="MaxRecentFilesTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&amp;Number of recent files:</string>
         </property>
         <property name="buddy">
          <cstring>MaxRecentFilesSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="MaxRecentFilesSpinBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>The maximum number of recent files to keep in menu</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="RandomizePercentTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&amp;Randomize factor:</string>
         </property>
         <property name="buddy">
          <cstring>RandomizePercentSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="RandomizePercentSpinBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Randomizer deviation from current value</string>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.1</double>
         </property>
         <property name="maximum">
          <double>100.0</double>
         </property>
         <property name="singleStep">
          <double>0.1</double>
         </property>
         <property name="value">
          <double>20.0</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="BaseFontSizeTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&amp;Base font size:</string>
         </property>
         <property name="buddy">
          <cstring>BaseFontSizeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="BaseFontSizeComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Base application font size (pt.)</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>(default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="StyleThemeTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&amp;Widget style theme:</string>
         </property>
         <property name="buddy">
          <cstring>StyleThemeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="StyleThemeComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Custom widget style theme</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>(default)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="ColorThemeTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&amp;Color palette theme:</string>
         </property>
         <property name="buddy">
          <cstring>ColorThemeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="ColorThemeComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Custom color palette theme</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>(default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wonton Soup</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KXStudio</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QToolButton" name="ColorThemeToolButton">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="toolTip">
          <string>Manage custom color palette themes</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MidiTabPage">
      <attribute name="title">
       <string>&amp;MIDI</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="MidiInputGroupBox">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Inputs</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QListWidget" name="MidiInputListView">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>MIDI Readable clients / Output ports</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="MidiOutputGroupBox">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Outputs</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QListWidget" name="MidiOutputListView">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>MIDI Writable clients / Input ports</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="8" margin="8"/>
 <tabstops>
  <tabstop>OptionsTabWidget</tabstop>
  <tabstop>ConfirmResetCheckBox</tabstop>
  <tabstop>ConfirmRemoveCheckBox</tabstop>
  <tabstop>CompletePathCheckBox</tabstop>
  <tabstop>MaxRecentFilesSpinBox</tabstop>
  <tabstop>RandomizePercentSpinBox</tabstop>
  <tabstop>BaseFontSizeComboBox</tabstop>
  <tabstop>StyleThemeComboBox</tabstop>
  <tabstop>ColorThemeComboBox</tabstop>
  <tabstop>ColorThemeToolButton</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qxgedit.qrc"/>
 </resources>
 <connections/>
</ui>
